openapi/components/schemas/Metadata/PaymentGatewayMetadata.yaml

```
@@ -117,6 +117,30 @@ properties:
     type: array
     items:
       type: string
+  supportedPaymentInstrumentSetupInstructions:
+    description: |
+      An array of the setup instructions supported by the gateway.
+    type: array
+    items:
+      type: string
+      enum:
+        - do-nothing
+        - authorize
+        - authorize-and-void
+        - sca
+      x-enumDescriptions:
+        authorize: |
+          Creates an `authorize` transaction in the amount/currency of the request.
+          If the gateway account is configured for strong customer authentication, it will be used.
+        authorize-and-void: |
+          Creates an `authorize` transaction in the amount/currency of the request, followed by a `void` (if the `authorize` was approved).
+          If the gateway account is configured for strong customer authentication, it will be used.
```

This looks excessive, as `sca` is just a different choice from these.
```suggestion
        authorize-and-void: |
          Creates an `authorize` transaction in the amount/currency of the request, followed by a `void` (if the `authorize` was approved).
```